labelbox/schema/dataset.py

```diff
@@ -52,40 +52,57 @@ class Dataset(DbObject, Updateable, Deletable):
     iam_integration = Relationship.ToOne("IAMIntegration", False,
                                          "iam_integration", "signer")
 
-    def create_data_row(self, **kwargs) -> "DataRow":
+    def create_data_row(self, items=None, **kwargs) -> "DataRow":
         """ Creates a single DataRow belonging to this dataset.
 
         >>> dataset.create_data_row(row_data="http://my_site.com/photos/img_01.jpg")
 
         Args:
+            items: Dictionary containing new `DataRow` data. At a minimum,
+                must contain `row_data` or `DataRow.row_data`.
             **kwargs: Key-value arguments containing new `DataRow` data. At a minimum,
                 must contain `row_data`.
 
         Raises:
+            InvalidQueryError: If both dictionary and `kwargs` are provided as inputs
             InvalidQueryError: If `DataRow.row_data` field value is not provided
                 in `kwargs`.
             InvalidAttributeError: in case the DB object type does not contain
                 any of the field names given in `kwargs`.
 
         """
+        invalid_argument_error = "Argument to create_data_row() must be either a dictionary, or kwargs containing `row_data` at minimum"
+
+        def convert_field_keys(items):
```

Comment:
I maybe move this out of the function scope and into module scope because this seems like a more general utility function type thing and doesn't need to be re-defined every time this method is called